storage/innobase/row/row0import.cc

```diff
@@ -2115,14 +2176,30 @@ dberr_t PageConverter::operator()(buf_block_t* block) UNIV_NOTHROW
 	return DB_SUCCESS;
 }
 
-/*****************************************************************//**
-Clean up after import tablespace. */
-static	MY_ATTRIBUTE((nonnull, warn_unused_result))
+static void reload_fts_table(row_prebuilt_t *prebuilt,
+                             dict_table_t* table)
+{
+
+  /* Reload the table in case of hidden fts column */
+  const table_id_t id= prebuilt->table->id;
+  prebuilt->table->release();
+  dict_sys.remove(prebuilt->table);
+  prebuilt->table= dict_table_open_on_id(
+                      id, true, DICT_TABLE_OP_NORMAL);
+  prebuilt->table->space= table->space;
+}
+
+/** Clean up after import tablespace.
+@param  prebuilt  prebuilt from handler
+@param  err       error code
+@param  table     table or fts hidden table
+@return error code or DB_SUCCESS */
+static
 dberr_t
 row_import_cleanup(
-/*===============*/
-	row_prebuilt_t*	prebuilt,	/*!< in/out: prebuilt from handler */
-	dberr_t		err)		/*!< in: error code */
+	row_prebuilt_t*	prebuilt,
+	dberr_t		err,
+	dict_table_t*	fts_table = nullptr)
```

Comment:
If you are reformatting the parameters, you could as well format them in the common style (no TABs, multiple parameters per line).

There is a mismatch between `@param table` and the parameter name `fts_table`. The comment does not really explain what the parameter is for.